### PR TITLE
Add Uninstall docs

### PR DESCRIPTION
**Pull request by BenWhitehead**

### CODE REVIEW

**Comment by BenWhitehead** [comment]
@ssk2 Please review. /cc @air

**Comment by air** [comment]
I think this is a little cryptic:
`Zookeeper Node provided to CASSANDRA_ZK`
`Using zkCli run the rmr command`

1. how do I find that value CASSANDRA_ZK
2. where can I find zkCli, on what machine do I run it, what rmr args do I need.

This is great btw, will serve as reference example for others.

File: docs/docs/uninstall.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+------------
+
+
+# Manual Uninstall on DCOS
+
+To Uninstall the Cassandra Mesos Framework cleanup the following items:
+
+1. Run `dcos package uninstall cassandra`
+2. Zookeeper Node provided to `CASSANDRA_ZK`
+    * Using zkCli run the `rmr` command
```
Comment (ssk2) [inline docs/docs/uninstall.md]: What path do we need to `rmr`?

File: docs/docs/uninstall.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+**DISCLAIMER**
+_This is a very early version of Cassandra-on-Mesos framework. This document, code behavior, and anything else may change without notice and/or break older installations._
+
+------------
+
+
+# Manual Uninstall on DCOS
+
+To Uninstall the Cassandra Mesos Framework cleanup the following items:
```
Comment (ssk2) [inline docs/docs/uninstall.md]: `cleanup` -> `complete`

Comment (ssk2) [inline docs/docs/uninstall.md]: `Uninstall` -> `uninstall`

Comment (ssk2) [inline docs/docs/uninstall.md]: `items` -> `actions`

File: docs/docs/uninstall.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+------------
+
+**DISCLAIMER**
+_This is a very early version of Cassandra-on-Mesos framework. This document, code behavior, and anything else may change without notice and/or break older installations._
+
+------------
+
+
+# Manual Uninstall on DCOS
```
Comment (ssk2) [inline docs/docs/uninstall.md]: Can we make `#Manual Uninstall` an additional H1 heading at the top and make this and the other heading H2 (i.e. `##`)?

Comment (ssk2) [inline docs/docs/uninstall.md]: (The title Markdown properties simply sets the HTML `<title>`.

File: docs/docs/uninstall.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+1. Run `dcos package uninstall cassandra`
+2. Zookeeper Node provided to `CASSANDRA_ZK`
+    * Using zkCli run the `rmr` command
+3. Remove the directory provided to `CASSANDRA_DATA_DIRECTORY`
+    * The default behavior is to write the Data into the Mesos Sandbox and does not need to be manually cleaned unless a specific value was provided.
+    
+    
+# Manual Uninstall on Mesos
+
+To Uninstall the Cassandra Mesos Framework cleanup the following items:
```
Comment (ssk2) [inline docs/docs/uninstall.md]: Same word changes as above.

File: docs/docs/uninstall.md (hunk 2 above)
Comment (ssk2) [inline docs/docs/uninstall.md]: Should we use "Service" instead of Framework here?

File: docs/docs/uninstall.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+1. Delete marathon app
+2. Shutdown framework
+    1. Get framework id from `http://mesos-master:5050/#/frameworks`
```
Comment (ssk2) [inline docs/docs/uninstall.md]: `id` - > `ID`

File: docs/docs/uninstall.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+# Manual Uninstall on DCOS
+
+To Uninstall the Cassandra Mesos Framework cleanup the following items:
+
+1. Run `dcos package uninstall cassandra`
+2. Zookeeper Node provided to `CASSANDRA_ZK`
+    * Using zkCli run the `rmr` command
+3. Remove the directory provided to `CASSANDRA_DATA_DIRECTORY`
+    * The default behavior is to write the Data into the Mesos Sandbox and does not need to be manually cleaned unless a specific value was provided.
```
Comment (ssk2) [inline docs/docs/uninstall.md]: `the Data` -> `data`

File: docs/docs/uninstall.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+To Uninstall the Cassandra Mesos Framework cleanup the following items:
+
+1. Delete marathon app
+2. Shutdown framework
+    1. Get framework id from `http://mesos-master:5050/#/frameworks`
+    2. run `curl -d "frameworkId=<framework_id_from_step_1>" http://mesos-master:5050/master/shutdown`
+3. Zookeeper Node provided to `CASSANDRA_ZK`
+    * Using zkCli run the `rmr` command
+4. Remove the directory provided to `CASSANDRA_DATA_DIRECTORY`
+    * The default behavior is to write the Data into the Mesos Sandbox and does not need to be manually cleaned unless a specific value was provided.
```
Comment (ssk2) [inline docs/docs/uninstall.md]: `the Data` -> `data`

File: docs/docs/uninstall.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+    
+# Manual Uninstall on Mesos
+
+To Uninstall the Cassandra Mesos Framework cleanup the following items:
+
+1. Delete marathon app
+2. Shutdown framework
+    1. Get framework id from `http://mesos-master:5050/#/frameworks`
+    2. run `curl -d "frameworkId=<framework_id_from_step_1>" http://mesos-master:5050/master/shutdown`
+3. Zookeeper Node provided to `CASSANDRA_ZK`
```
Comment (ssk2) [inline docs/docs/uninstall.md]: `Zookeeper` -> `ZooKeeper`

File: docs/docs/uninstall.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+
+------------
+
+
+# Manual Uninstall on DCOS
+
+To Uninstall the Cassandra Mesos Framework cleanup the following items:
+
+1. Run `dcos package uninstall cassandra`
+2. Zookeeper Node provided to `CASSANDRA_ZK`
```
Comment (ssk2) [inline docs/docs/uninstall.md]: `Zookeeper` -> `ZooKeeper`

File: docs/docs/uninstall.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+**DISCLAIMER**
+_This is a very early version of Cassandra-on-Mesos framework. This document, code behavior, and anything else may change without notice and/or break older installations._
+
+------------
+
+
+# Manual Uninstall on DCOS
+
+To Uninstall the Cassandra Mesos Framework cleanup the following items:
+
```
Comment (ssk2) [inline docs/docs/uninstall.md]: Worth pointing out that this will _completely_ uninstall the Cassandra-on-Mesos framework - data and all.

**Comment by BenWhitehead** [comment]
@ssk2 I incorporated all your proposed edits and added some of the notes you mentioned. Please review.

@air In the case of DCOS I've added in the default values for the `rmr` command.  The point about how to find the values I think plays into "How do we expose configuration values for Services", WDYT?

File: docs/docs/uninstall.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+**DISCLAIMER**
+_This is a very early version of Cassandra-on-Mesos framework. This document, code behavior, and anything else may change without notice and/or break older installations._
+
+------------
+
+
+## Manual Uninstall on DCOS
+
+_All instructions below assume the default values were used when installed on DCOS_
```
Comment (ssk2) [inline docs/docs/uninstall.md]: What's happening with the L in below here/

File: docs/docs/uninstall.md [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+    * The default behavior is to write data into the Mesos Sandbox and does not need to be manually cleaned unless a specific value was provided.
+    
+    
+## Manual Uninstall on Mesos
+
+_Following the instructions will completely uninstall the Cassandra-on-Mesos framework -- data and all, no recovery will be possible._
+
+To uninstall the Cassandra Mesos Framework complete the following actions:
+
+1. Delete marathon app
```
Comment (ssk2) [inline docs/docs/uninstall.md]: `marathon` -> `Marathon`

**Comment by ssk2** [comment]
One small correction, otherwise good!

**Pull request merged**